aptos-move/framework/aptos-framework/sources/multisig_account.move

```diff
@@ -875,7 +917,14 @@ module aptos_framework::multisig_account {
             table::contains(&multisig_account_resource.transactions, sequence_number),
             error::not_found(ETRANSACTION_NOT_FOUND),
         );
+
+        let transaction = table::borrow(&multisig_account_resource.transactions, sequence_number);
+        let voted = simple_map::contains_key(&transaction.votes, &address_of(owner));
         let (_, num_rejections) = remove_executed_transaction(multisig_account_resource);
+        // In context, treat the owner's absence of vote as an implicit rejection.
+        if (!voted) {
+            num_rejections = num_rejections + 1;
+        };
```

Comment:
I suggest:

```move
// In context, treat the owner's absence of vote as an implicit rejection.  
if (!voted) {  
    num_rejections = num_rejections + 1;  
} else {  
    // In context, assume that if they have voted, they now want their vote to be rejection.  
    let approved = *simple_map::borrow(&transaction.votes, &owner);  
    if (approved) num_rejections = num_rejections + 1;  
}; 
```

Reply:
Updated to call `reject_transaction` so that it overrides the previous vote and emit an event accordingly.